### PR TITLE
ANSI -> Wide

### PR DESCRIPTION
**Pull request by hypersw**

### CODE REVIEW

**Comment by dblock** [comment]
Thanks! Can you please add a line to CHANGELOG for this? Also a test could be fun to write for it, but I'll merge without too, it's a pretty innocent change.

**Comment by dblock** [comment]
Can you squash your commits please? Also add a note when you're done, Github doesn't send notifications for additional changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * [#16](https://github.com/dblock/resourcelib/issues/16): Fixed NuGet error "Package restore is disabled by default." when building by updating to NuGet 2.8.1 - [@icnocop](https://github.com/icnocop).
 * Including Vestris.ResourceLib.pdb and Vestris.ResourceLib.xml in release zip for documentation and easier debugging support - [@icnocop](https://github.com/icnocop).
 * [CI](https://ci.appveyor.com/project/dblock/resourcelib): Setup CI on AppVeyor - [@dblock](https://github.com/dblock).
+* [#26](https://github.com/dblock/resourcelib/pull/26): Fixed opening PE files at non-ANSI paths by changing from ANSI to Wide version of PInvoke - [@hypersw](https://github.com/hypersw)
```
Comment (dblock) [inline CHANGELOG.md]: My ADD: Missing a period at the end, please :)

**Comment by dblock** [comment]
Bump @hypersw

**Comment by hypersw** [comment]
I think I've implemented most of the requests, except for the squash stuff which I usually don't do, so I got no consideration for the target setup. How would you like to see them squashed?

**Comment by dblock** [comment]
You're right, this is good, merging.

**Pull request merged**